lib/plant.py

```diff
@@ -1,4 +1,23 @@
+import time
+import board
+import busio
+from adafruit_seesaw.seesaw import Seesaw
+
 class Plant():
 
     def soil_moisture(self):
-        return 350
+
+        i2c_bus = busio.I2C(board.SCL, board.SDA)
```

Comment:
Is busio specific to soil_moisture, or could this be added to the __init__ for plant?

Reply:
I'm not sure. Will try to find out